common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
+info:
+  title: Generieke datatypen voor Gemeentelijke APIs onder regie van VNGRealistaie
+  description: |
+    De definitie van generieke datatypen die worden gebruikt door gemeentelijke API's.
+    Deze datatypen komen overeen met het basismodel Generieke Datatype Gemeenten waarnaar verwezen wordt vanuit informatiemodellen.
+    Er worden uit dit bestand met herbruikbare componenten geen componenten verwijderd.
+    Als een component in de toekomst niet meer hergebruik mag worden dan wordt het deprecated verklaard.
+  version: '1.0.0'
+  contact:
+    name: Generieke Datatypen Gemeenten.
+    url: https://github.com/VNG-Realisatie/API-Kennisbank/Common
+  license:
+    name: European Union Public License, version 1.2 (EUPL-1.2)
+    url: https://eupl.eu/1.2/nl/
+paths: {}
+components:
+  schemas:
+    UUID:
+      type: "string"
+      minLength: 36
+      maxLength: 36
+      example: "123e4567-e89b-12d3-a456-426614174000"
+    OIN:
+      type: "string"
+      minLength: 20
+      maxLength: 20
+      pattern: ^[0-9]{20}$
+      description: |
+                    Het Organisatie-identificatienummer (OIN) is een uniek nummer dat Logius kan toekennen aan organisaties om zich te kunnen identificeren, authentiseren en of autoriseren bij digitaal berichtenverkeer binnen en met de overheid.
+    DatumOnvolledig:
+      type: "object"
+      description: |
+                    Gegevens over de datums die mogelijk niet volledig zijn.
+      properties:
+        dag:
+          type: "integer"
+          description: |
+                        Als de dag van de datum bekend is wordt dit element gevuld, ook als de volledige datum bekend is.
+          minimum: 1
+          maximum: 31
+          example: 3
+        jaar:
+          type: "integer"
+          description: |
+                        Als het jaar van de datum bekend is wordt dit element gevuld, ook als de volledige datum bekend is.
+          maximum: 9999
+          example: 1989
+        maand:
+          type: "integer"
+          description: |
+                        Als de maand van een datum bekend is wordt dit element gevuld, ook als de volledige datum bekend is.
+          minimum: 1
+          maximum: 12
+          example: 5
+        datum:
+          type: "string"
+          description: |
+                        Als de volledige datum bekend is wordt de datum gevuld die in de date definitie past.
+          format: "date"
+          example: "1989-05-03"
```

Comment:
Het attribute 'datum' ontbreekt helemaal in het SIM model. Voeg het daar ook toe.

Reply:
Done